Lib/annotationlib.py

```diff
@@ -459,13 +495,15 @@ def unary_op(self):
 
 
 class _StringifierDict(dict):
-    def __init__(self, namespace, globals=None, owner=None, is_class=False):
+    def __init__(self, namespace, globals=None, owner=None, is_class=False, create_unions=False):
```

Comment:
It looks like with your solution you're not using `create_unions` any more but it carried over from my changes.

Reply:
Thanks, will clean it up